celery/contrib/rdb.py

```diff
@@ -124,6 +124,7 @@ def get_avail_port(self, host, port, search_limit=100, skew=+0):
         this_port = None
         for i in range(search_limit):
             _sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+            _sock.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
```

Comment:
`SO_REUSEADDR` has been around since time immemorial. You're probably looking at `SO_REUSEPORT`, which was added in 3.9.